src/lsp/cobol_parser/cobol_parser.ml

```diff
@@ -18,33 +18,18 @@ module PTree = PTree
 module PTree_visitor = PTree_visitor
```

Comment:
Not in the patch, but why does `cobol_parser` exports a visitor interface ? For me, there should be a library `cobol_visitors`, and `cobol_ast` and `cobol_parser` should be independent of it, i.e. it should be possible to use the parser without using the visitors.

Reply:
Yes that's a change planned for a little later.